dapps/fast-usdc/vm-integration/test/fast-usdc.test.ts

```diff
@@ -43,6 +54,7 @@ const {
 } = process.env;
 
 test.before('bootstrap', async t => {
+  // TODO move out of agoric-sdk
```

Comment:
a better option may be to rename it `decentral-itest-plain-orchestration-config` because it's just the orch config minus chain config.

alternately, call this minimal one `orch-config` and the bigger one `orch-and-chain-config`.